src/Commands/AdminCommands/ChatsCommand.php

```diff
@@ -58,6 +58,7 @@ public function execute()
         $results = DB::selectChats(
```

Comment:
Well, I think I have no clue how to do this in a clean way, I would have to declare those somewhere, but where to make them accessible at all time?